learning-another-language/intro-to-functions.md

```diff
@@ -0,0 +1,116 @@
+# Intro to Functions
+
+## Introduction
+
+There are _many_ ways to define functions. For now, we will dive into two ways:
+
+- Functional expressions
+- Function declarations
+
+Both of these methods have the same result: defining a function.
+
+The most important difference between them is the context that we typically use them in. There are technical differences between them, but these are not relevant on the introductory level.
+
+## Functional Expressions
+
+The name "functional expression" refers to the function acting as an expression that a variable references.
+
+Let's look at this code, which defines a function named `increaseQuantity`:
+
+<!-- prettier-ignore-start -->
+```js
+const increaseQuantity = function(initial_qty) {
+    return initial_qty + 1;
+}
+```
+<!-- prettier-ignore-end -->
+
+Let's break down the details of this `increaseQuantity` function:
+
+- We create a `const` variable named our function name, `increaseQuantity`.
+- We assign `increaseQuantity` to the right-hand value.
```

Comment:
I'm gonna remember the "assign" directionality thing eventually

Reply:
Another option is to use "set" instead of "assign". If you prefer to phrase things from the perspective of the thing on the left (perfectly natural, since we encounter that first), then "set" has both the correct ordering and the correct directionality.

We set `increaseQuantity` to the right-hand value.

Is totally correct.